adafruit_bluefruit_connect/button_packet.py

```diff
@@ -35,7 +35,25 @@
 from .packet import Packet
 
 class ButtonPacket(Packet):
-    """A packet containing a button name and its state"""
+    """A packet containing a button name and its state."""
+
+    B_1 = '1'
```

Comment:
If we need to call it something more than `1`, then I think we should call it `BUTTON_1`. It's clearer that way.

Reply:
Should we call `UP`, etc. `BUTTON_UP`? Originally I did `ONE`, `TWO`, but that seemed kind of silly. `B1` was not accepted by pylint.

Reply:
Hmm. I feel like `UP` and `DOWN` etc. are clear. But perhaps for consistency?